test-app/.stylelintignore

```diff
@@ -0,0 +1,8 @@
+# unconventional files
+/blueprints/*/files/
+
```

Comment:
Does this need to ignore node_modules or does it by default?

Reply:
The default configuration seems to be ignoring this by default, but I couldn't find any reference to it in the documentation. In the base `ember new` template, there's no line for node_modules in the .styleignore file.